tests/integration_tests/functional/test_logging.py

```diff
@@ -59,7 +59,9 @@ def check_log_message_format(log_str, instance_id, level, show_level, show_origi
     pattern += "\\].*"
 
     mo = re.match(pattern, tag_and_msg)
-    assert mo is not None
+    assert (
+        mo is not None
+    ), f"Log message ({tag_and_msg}) does not match pattern ({pattern})."
```

Comment:
Isn't this redundant, since all the local variables are printed?

Reply:
If we already print all local variables it does allow similar functionality but I prefer this approach as it makes it clearer (rather than needing to look through a larger portion of text).